lib/biokbase/catalog/registrar.py

```diff
@@ -663,17 +663,17 @@ def run_docker_container(self, dockerclient, image_name, token,
                 self.log("    Docker command: " + command)
             dockerclient.start(container=cnt_id)
             stream = dockerclient.logs(container=cnt_id, stdout=True, stderr=True, stream=True)
-            line = ""
-            for char in stream:
+            line = []
+            for char.decode('utf-8') in stream:
```

Comment:
Well, I'm pretty sure you can't use that syntax. https://travis-ci.org/kbase/catalog/builds/518851015?utm_source=github_status&utm_medium=notification

Even if you could, this would fail if the generator returns a unicode string which is what it ought to be doing according to it's API https://docker-py.readthedocs.io/en/stable/api.html#docker.api.container.ContainerApiMixin.logs